### PR TITLE
Remove VirusTotal references

### PR DESCRIPTION
**Pull request by JuanGarriuz**
## Description

Removed VirusTotal references

image

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/compliance/nist/malware-detection.rst
Comment (guidomodarelli) [inline source/compliance/nist/malware-detection.rst:14]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

File: source/compliance/hipaa/malware-detection.rst
Comment (guidomodarelli) [inline source/compliance/hipaa/malware-detection.rst:20]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

File: source/getting-started/index.rst
Comment (guidomodarelli) [inline source/getting-started/index.rst:60]: Why were these lines changed to "Endpoints summary"?
> [!NOTE]
> The "Endpoints Summary" no longer exists; it has been replaced by "Agents Management » Summary."

File: source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst
Comment (guidomodarelli) [inline source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

**Review by guidomodarelli — CHANGES_REQUESTED** [review]

**Review by javimed — CHANGES_REQUESTED** [review]
Hi @JuanGarriuz . Please consider this:

- There's a compilation warning about a broken reference that needs to be fixed: `source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43: WARNING: unknown document: '/user-manual/capabilities/malware-detection/virus-total-integration'`
- Virustotal related documentation stays for older versions. Therefore, you souldn't remove redirectioning in older versions, just adapt redirections concerning 4.10 <-> 4.9. Check the code block below. I can help twith it.
- Your teammates must approve the PR once they consider it's ready for merging.

```
...
/* *** RELEASE 4.10 ****/

/* Redirections from 4.9 to 4.10  */

redirections.push(
  {
    'target': ['4.9=>4.10', '4.10=>4.9'],
    ...
  }

/* Pages no longer available in 4.10 */

removedUrls['4.10'] = [
...
]
...
```

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
TEST: ✅ 

Captura de pantalla 2024-11-07 a la(s) 7 16 04 a  m

**Comment by JuanGarriuz** [comment]
> Hi @JuanGarriuz . Please consider this:
> 
> * There's a compilation warning about a broken reference that needs to be fixed: `source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43: WARNING: unknown document: '/user-manual/capabilities/malware-detection/virus-total-integration'`
> * Virustotal related documentation stays for older versions. Therefore, you souldn't remove redirectioning in older versions, just adapt redirections concerning 4.10 <-> 4.9. Check the code block below. I can help twith it.
> * Your teammates must approve the PR once they consider it's ready for merging.
.....
> ```

After discussing it with the team, those changes will be added to the " Update Malware detection documentation to include VirusTotal" issue.

**Review by javimed — APPROVED** [review]

**Pull request merged**
